frontend/src/components/post-card.tsx

```diff
@@ -34,7 +34,7 @@ export default function PostCard({ post, testId = 'postcard' }: { post: Post } &
           <p className="line-clamp-2 text-sm text-light-description dark:text-dark-description">
             {post.description}
           </p>
-          <div className="mt-4 flex flex-wrap gap-2">
+          <div className="mt-4 flex gap-2">
```

Comment:
Why are we touching this? Is it any related to the issue that you are working? because this may effect in the responsiveness

Reply:
Yea. As it is given flex-warp, it will grow to the next line and the height for this one will be increased. So removing flex-wrap, it will always be in one line, be it as many tags as possible. Like this :
![Screenshot 2024-05-22 095158](https://github.com/krishnaacharyaa/wanderlust/assets/107207581/c499866a-75fa-463c-8ef7-c5b7a6e890f9)

Reply:
Very interesting @Alen-Scaria1516. Now this will be challenging, can we aim on the things which gets fit perfectly let it show and only the right most one will be ... and then rest cut off.

Meaning. here on the above example instead of it looking like this, can we aim at..

Thailand | Andaman & nicobar | Niger... The rest M... B... C... F... is not show at all in this card. 
Can you try this?

Reply:
Oh ok. That would be quite hard. But lemme try 🙂
Responsive wise there won't be any issue. You can see this from the following demo 

https://github.com/krishnaacharyaa/wanderlust/assets/107207581/f1b117c3-83e6-4bfa-9c39-469a39f6b182

Reply:
> Oh ok. That would be quite hard. But lemme try 🙂 Responsive wise there won't be any issue. You can see this from the following demo
> 
>  wanderlust01.mp4

Agreed 💯. But give it a try!!